Source/Core/VideoBackends/D3D/main.cpp

```diff
@@ -63,16 +88,13 @@ void VideoBackend::FillBackendInfo()
   g_Config.backend_info.bSupportsClipControl = true;
   g_Config.backend_info.bSupportsDepthClamp = true;
   g_Config.backend_info.bSupportsReversedDepthRange = false;
-  g_Config.backend_info.bSupportsLogicOp = true;
   g_Config.backend_info.bSupportsMultithreading = false;
   g_Config.backend_info.bSupportsGPUTextureDecoding = true;
-  g_Config.backend_info.bSupportsST3CTextures = false;
```

Comment:
We still want to reset this to false, in the case that the device isn't created below and we're switching from a different backend. FWIW, the values here aren't used unless a game is running, so skipping them is okay, but we'll set them properly when we rewrite this to remove the global state.

Reply:
Don't worry about those. They're being unconditionally assigned to later down this function, so I just removed the first writes as they'll be overwritten anyway!

Reply:
They're not assigned unconditionally if the device is null (i.e. we're not booting a game).